File: app/src/main/java/edu/gatech/cs2340/Bank.java
package edu.gatech.cs2340;

public class Bank {
    private int bankCredits = 0;
    private double interest = .05;

    void deposit(int amount) {
        bankCredits += amount;
    }

    void withdraw(int amount) {
        bankCredits -= amount;
    }

    void addInterest() {
        bankCredits += bankCredits * interest;
    }

    int getBankCredits() {
        return bankCredits;
    }

}

File: app/src/main/java/edu/gatech/cs2340/Bank.kt
package edu.gatech.cs2340

class Bank {
    var bankCredits = 0
    private val interest = .05

    fun deposit(amount: Int) {
        bankCredits += amount
    }

    fun withdraw(amount: Int) {
        bankCredits -= amount
    }

    fun addInterest() {
        bankCredits += (bankCredits * interest).toInt()
    }

}
